package.json

```diff
@@ -7,18 +7,20 @@
     "start": "next start"
   },
   "dependencies": {
+    "faker": "^5.1.0",
+    "install": "^0.13.0",
     "next": "10.0.1",
+    "npm": "^6.14.8",
```

Comment:
Have you used npm instead of yarn? Please use yarn add [package] to add new dependencies.

Reply:
Find the docs of yarn here: https://yarnpkg.com/ 
Don't hesitate to ask any upcoming questions.